src/vs/workbench/api/browser/mainThreadDebugService.ts

```diff
@@ -223,7 +223,14 @@ export class MainThreadDebugService implements MainThreadDebugServiceShape, IDeb
 				}));
 				this.debugService.addBreakpoints(uri.revive(dto.uri), rawbps);
 			} else if (dto.type === 'function') {
-				this.debugService.addFunctionBreakpoint(dto.functionName, dto.id, dto.mode);
+				this.debugService.addFunctionBreakpoint({
```

Comment:
This seems fine, right @connor4312? I don't know why it would have been limited

Reply:
yea, I don't either. This limitation dates from https://github.com/microsoft/vscode/commit/eba681fed8b36e9abfeb0b80c7b0c5078c7d843d and I think it's just a bug.